src/essays/graph-experience-learning/index.md

```diff
@@ -0,0 +1,147 @@
+
+### What's a Filter?
+Think search engine, or a menu, and you're trying to narrow down your options from A Lot of Stuff.
+
+## A Practical Example
+Let's say we're examining a service that makes bubble tea. The factors in this example will be variety of leaf (green or black), firmness of bubbles (hard or soft), and addons (sweetened or spiced). In this example, there are 8 complete permutations:
+
+- green soft sweetened
+- black soft sweetened
+- green hard sweetened
+- black hard sweetened
+- green soft spiced
+- black soft spiced
+- green hard spiced
+- black hard spiced
+
+
+So you're an end user looking at options, playing with the options available harmlessly starting with a black tea, but no bubbles, sugar, or spices. Removing an item is simple enough. Typically not specifying will also return results that include sugar or spice simply because you didn't specify that you wanted either. (this is a classic problem and is related to English having an ambiguous meaning to the word "or")
+
+#### 23:00
+Then you think to yourself "But I want lemon" Lemon in this fictional machine's mind does not exist and therefore is not a clickable option. Perhaps you even type in lemon in a freeform text field, but the gold standard of typing in text that does not match any entries is to report just that; no entries found. You're at an impass. The site tried to provide you with a list of results but came back with nothing. What if in place of nothing, you are prompted "What's a lemon?"
+- variety of tea
+- a way to prepare bubbles
+- an addon
+- something new
+The user may think "Why don't I make this a new category called `fruit`. Sure. But this is strange... why is it asking me?" (copywriting goes a long way. Learning by doing goes farther.) Upon submitting, the service as whole, persons and machine alike, get to learn three things:
+1. People may want lemon in their tea
+2. This person thought that there should be a new category called `fruit`
+3. If someone is ordering black tea, that person might also want lemon
+
+## Example 1 Redux 21:00
+Learning is all about drawing connections. "These three points are connected" You can have lemon in your database without presenting it to your end users. You can have everysuggestion every person ever made, someone might suggest dirt at some point. Will you add dirt to the menu? No! But lemon probably will continue to present itself. With enough agreeance that fruit deserves a place in how ingredients are divided, fruit could in this example be added, and while we're at it, let's throw in oranges. (19:44) In a traditional experience, at the point of making no possible combination, you'd leave the site, maybe write the company an email, maybe tweet about it.
```

Comment:
`You can have everysuggestion every person ever made, someone might suggest dirt at some point.` (two separate sentences)